src/openms/include/OpenMS/FILTERING/DATAREDUCTION/MassTraceDetection.h

```diff
@@ -105,7 +105,7 @@ namespace OpenMS
         typedef std::multimap<double, std::pair<Size, Size> > MapIdxSortedByInt;
 
         /// The internal run method
-        void run_(const MapIdxSortedByInt& chrom_apices,
+        void run_(const std::vector<std::pair<double,std::pair<Size,Size>>>& chrom_apices,
```

Comment:
would a std::tuple be cleaner?

Reply:
by all means, please use a `struct` with named parameters